daemon/graphdriver/overlay/overlay.go

```diff
@@ -195,10 +211,12 @@ func (d *Driver) GetMetadata(id string) (map[string]string, error) {
 	return metadata, nil
 }
 
+//Cleanup simply returns nil and do not change the existing filesystem.
```

Comment:
Should add `This is required to satisfy the graphdriver.Driver interface` (or is it ProtoDriver?)